public/app/plugins/panel/canvas/ConnectionAnchors.tsx

```diff
@@ -136,8 +140,8 @@ const getStyles = (theme: GrafanaTheme2) => ({
     cursor: cursor;
     position: absolute;
     pointer-events: auto;
-    width: 16px;
-    height: 16px;
+    width: calc(${connectionAnchorHighlightOffset}px * 2);
+    height: calc(${connectionAnchorHighlightOffset}px * 2);
```

Comment:
Not sure this should be scaled / dependent on the offset 🤔 I think we can revert this back to 16px / 16px